spinalcordtoolbox/deepseg/models.py

```diff
@@ -356,6 +365,17 @@
                              'resolution, for images with another resolution divide 30/your_axial_resolution.',
          'url': 'https://github.com/ivadomed/model_seg_ms_mp2rage',
          'models': ['model_seg_ms_lesion_mp2rage']},
+    'model_seg_canal_t2w':
+        {'description': 'Segmentation of spinal canal on T2w contrast',
+         'long_description':'This model segments the spinal canal, or in an anatomic definition the dural sac, on T2w contrast.'
```

Comment:
There are some code quality issues detected in https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/11746433653/job/32726178139?pr=4687. Could you please make sure the check passes? Thank you! :)

Reply:
Done! I didn't have flake8 on my new laptop I just downloaded it, it should not appear again.